analyzer/javacg-opal/src/test/java/eu/fasten/analyzer/javacgopal/OPALPluginTest.java

```diff
@@ -103,19 +103,19 @@ public void testFileNotFoundException() {
         assertEquals(FileNotFoundException.class.getSimpleName(), error.getClass().getSimpleName());
     }
 
-    @Test
-    public void testEmptyCallGraph() throws JSONException {
-        JSONObject emptyCGCoordinate = new JSONObject("{\n"
-                + "    \"groupId\": \"activemq\",\n"
-                + "    \"artifactId\": \"activemq\",\n"
-                + "    \"version\": \"release-1.5\",\n"
-                + "    \"date\":\"1574072773\"\n"
-                + "}");
-
-        plugin.consume(emptyCGCoordinate.toString());
-        assertFalse(plugin.produce().isPresent());
-        assertEquals(EmptyCallGraphException.class, plugin.getPluginError().getClass());
-    }
+    /* @Test */
```

Comment:
Why was the test commented out?

Reply:
https://github.com/fasten-project/fasten/pull/91/commits/f3bd73cc723b135fd753935a90514241c518673a